[2024-05-27-16:37:11.751] HTTPConnectionPool(host='192.168.3.52', port=8001): Max retries exceeded with url: /sign-firmware-development (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x000001D81A0D9A90>, 'Connection to 192.168.3.52 timed out. (connect timeout=3)'))